docs/layout-docs.html

```diff
@@ -35,7 +35,51 @@
     />
     <link href="https://fonts.googleapis.com/css2?family=Sen:wght@400;700&amp;display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="../scss/fermyon.scss" />
-
+    <script>
```

Comment:
Do we want to pull this script into a file rather than duplicating it in all layouts?

Reply:
Scripts in `<head>` block first render until they are loaded so there is some marginal benefit in keeping them inline. Ideally they'd be inlined by your build tooling but I think thats a question for @flynnduism

Reply:
Ah, thanks for the explanation.
I think we can merge and explore whether we want to somehow use tooling to inline them later.